scripts/install_zds.sh

```diff
@@ -550,25 +550,36 @@ if  ! $(_in "-data" $@) && ( $(_in "+data" $@) || $(_in "+base" $@) || $(_in "+f
         exit 1
     fi
 
-    python manage.py loaddata fixtures/*.yaml; exVal=$?
-
-    python manage.py load_factory_data fixtures/advanced/aide_tuto_media.yaml; exVal=($exVal + $?)
-
+    settingsParams=""
     if $(_in "--travis-output" $@); then
-        python manage.py load_fixtures --size=low --all --settings zds.settings.travis_fixture; exVal=($exVal + $?)
-    else
-        python manage.py load_fixtures --size=low --all; exVal=($exVal + $?)
+        settingsParams="--settings=zds.settings.travis_fixture"
+        python manage.py migrate ${settingsParams}; exVal=$?
+
+        if [[ $exVal != 0 ]]; then
+            print_error "Warning: Cannot make migration for travis script"
+        fi
     fi
 
+    python manage.py loaddata fixtures/*.yaml ${settings_params}; exVal=$?
+    python manage.py load_factory_data fixtures/advanced/aide_tuto_media.yaml ${settings_params}; exVal=($exVal + $?)
+    python manage.py load_fixtures --size=low --all ${settings_params}; exVal=($exVal + $?)
+
+    futureExit=false
     if [[ $exVal != 0 ]]; then
         print_error "!! Cannot generate-fixtures (use \`-data\` to skip)"
+        futureExit=true
         # don't exit here, because we have to stop zmd !
     fi
 
     make zmd-stop; exVal=$?
 
     if [[ $exVal != 0 ]]; then
         print_error "Warning: Cannot stop zmd"
+
+    fi
+
+    if [[ futureExit ]]; then
+        exit 1
```

Comment:
[Je suis con sur ce coup là](https://github.com/zestedesavoir/zds-site/commit/c90e2b5f80e100535b296da73adfff1082aec4db). J'ai complètement zappé que sans code erreur travis n'allait pas planter -.-" Tu peux rajouter un commentaire : `# Throw error for travis` ?